packages/shell/src/cli-run.ts

```diff
@@ -43,6 +43,7 @@ export async function run(options: { env?: string; databaseURL?: string; apiKey?
   const defaultEval = getDefaultEval();
   const replServer = repl.start({
     preview: true,
+    ignoreUndefined: true,
```

Comment:
Including also this, unrelated to the main PR goal

Reply:
🤔 I played with that option and I preferred to leave it disabled. The reason is that when you press enter multiple times you just see a list of prompts (`>`) and it's confusing because you don't know if there's a command waiting to be closed (a missing `)` or `}`, etc.) or it's just that there's nothing to execute.